components/network/openvpn/patches/03-openvpn-man8.patch

```diff
@@ -1,101 +1,89 @@
---- openvpn-2.4.6/doc/openvpn.8	2018-04-24 09:12:55.000000000 +0000
-+++ openvpn-2.4.6/doc/openvpn.8	2019-02-19 18:28:35.008492638 +0000
-@@ -41,7 +41,7 @@
- .  TP \\$1\" no doublequotes around argument!
- ..
- .\" End of TQ macro
--.TH openvpn 8 "28 February 2018"
-+.TH openvpn 1m "28 February 2018"
- .\"*********************************************************
+diff -Nru openvpn-2.5.7.orig/doc/openvpn.8 openvpn-2.5.7/doc/openvpn.8
```

Comment:
Please remove this man8 patch completely. With the merge of IPD4 illumos moved many man pages to locations where other OS's have them.